components/website/utils/Schedule/Table/Block/index.jsx

```diff
@@ -3,65 +3,67 @@ import Image from 'next/image';
 import Link from 'next/link';
 import { useEffect } from 'react'
 
-export default function Block(props) {
+function BlockItem({ id, coffeeBreak, startTime, endTime, activityType, summary, 
+    author, description, focused, location, detailed, hyperlink}) {
+     
+    const ident = id;
+    console.log(ident);
 
-    const id = `${props.date}-${props.id}`;
+    useEffect(() => {
+        const b = document.getElementById(ident);
```

Comment:
Why not use id directly, since it is the prop? 
```suggestion
        const b = document.getElementById(id);
```